### ISSUE TITLE
Pivot directive needs more tests to increase code coverage

### ISSUE THREAD

**Issue by andrewconnell** [ngOfficeUIFabric/ng-officeuifabric#321]
As mentioned in #33...

We have an issue with this directive. Code coverage is showing below 50% for tests... see this build report: 

https://circle-artifacts.com/gh/ngOfficeUIFabric/ng-officeuifabric/115/artifacts/0/tmp/circle-artifacts.UPLlyLE/code-coverage/index.html

We need a lot more tests to cover the code in this directive.

**Comment by jjczopek** [comment]
While developing, I have added more than 20 test cases to test each aspect of the directive. However while running the tests, most of the parts of the code look to be not accessed at all, but the test cases don't fail.

As an example, the directive has a link function. This function should be called at least once for every test case, because instance of the directive is created before each test case. But when you look into the code coverage report. it is called only once:
![image](https://cloud.githubusercontent.com/assets/865473/14893078/3f895538-0d6e-11e6-9e2f-e869ea5e8690.png)

There are test cases for validating attributes and they pass, but the calls to respective functions are not recorded :/ `getClasses` function is called at least twice because there are test cases checking for the classes: 
- [should render large pivot](https://github.com/ngOfficeUIFabric/ng-officeuifabric/blob/master/src/components/pivot/pivotDirective.spec.ts#L42)
- [should be rendered as tabs](https://github.com/ngOfficeUIFabric/ng-officeuifabric/blob/master/src/components/pivot/pivotDirective.spec.ts#L89)

I don't think I did anything extra-ordinary compared to other directives I have implemented, but only this one got this weird coverage issue :/
